library/std/tests/sync/lazy_lock.rs

```diff
@@ -165,3 +165,32 @@ fn lazy_force_mut() {
     p.clear();
     LazyLock::force_mut(&mut lazy);
 }
+
+/// Verifies that when a `LazyLock` is poisoned, it panics with the correct error message ("LazyLock
+/// instance has previously been poisoned") instead of the underlying `Once` error message.
+#[test]
+#[cfg_attr(not(panic = "unwind"), ignore = "test requires unwinding support")]
+#[should_panic(expected = "LazyLock instance has previously been poisoned")]
```

Comment:
The reason this test passes is because `#[should_panic(expected = "...")]` likely checks the last panic message, not the first.